cid/cli.py

```diff
@@ -137,51 +140,66 @@ def delete(ctx, dashboard_id, **kwargs):
 @cid_command
 def update(ctx, dashboard_id, force, recursive, **kwargs):
     """Update Dashboard"""
     ctx.obj.update(dashboard_id, force=force, recursive=recursive, **kwargs)
 
 
-@click.option('-v', '--verbose', count=True)
-@click.option('-y', '--yes', help='confirm all', is_flag=True, default=False)
-@click.option('--dashboard-id', help='QuickSight dashboard id', default=None)
+@click.option("-v", "--verbose", count=True)
+@click.option("-y", "--yes", help="confirm all", is_flag=True, default=False)
+@click.option("--dashboard-id", help="QuickSight dashboard id", default=None)
 @cid_command
 def open(ctx, dashboard_id, **kwargs):
     """Open Dashboard in browser"""
     ctx.obj.open(dashboard_id, **kwargs)
 
 
-@click.option('-v', '--verbose', count=True)
-@click.option('-y', '--yes', help='confirm all', is_flag=True, default=False)
+@click.option("-v", "--verbose", count=True)
+@click.option("-y", "--yes", help="confirm all", is_flag=True, default=False)
 @cid_command
 def cleanup(ctx, **kwargs):
     """Delete unused resources (QuickSight datasets, Athena views)"""
     ctx.obj.cleanup(**kwargs)
 
 
-@click.option('-v', '--verbose', count=True)
-@click.option('-y', '--yes', help='confirm all', is_flag=True, default=False)
-@click.option('--dashboard-id', help='QuickSight dashboard id', default=None)
-@click.option('--share-method', help='Sharing method', default=None, type=click.Choice(['folder', 'user', 'account']))
-@click.option('--folder-method', help='Folder to use', default=None, type=click.Choice(['new', 'existing']))
-@click.option('--folder-id', help='QuickSight folder id (existing)', default=None)
-@click.option('--folder-name', help='QuickSight folder name (new)', default=None)
-@click.option('--quicksight-user', help='QuickSight user', default=None)
+@click.option("-v", "--verbose", count=True)
+@click.option("-y", "--yes", help="confirm all", is_flag=True, default=False)
+@click.option("--dashboard-id", help="QuickSight dashboard id", default=None)
+@click.option("--share-method", help="Sharing method", default=None, type=click.Choice(["folder", "user", "account"]))
+@click.option("--folder-method", help="Folder to use", default=None, type=click.Choice(["new", "existing"]))
+@click.option("--folder-id", help="QuickSight folder id (existing)", default=None)
+@click.option("--folder-name", help="QuickSight folder name (new)", default=None)
+@click.option("--quicksight-user", help="QuickSight user", default=None)
 @cid_command
 def share(ctx, dashboard_id, **kwargs):
     """Share QuickSight resources (Dashboard, Datasets, DataSource)"""
-    
+
     ctx.obj.share(dashboard_id)
 
-if __name__ == '__main__':
+
+@click.option("-v", "--verbose", count=True)
+@click.option("-y", "--yes", help="confirm all", is_flag=True, default=False)
+@click.option("--qs-notification-email", help="QuickSight Notification Email", default="")
```

Comment:
please choose if you use click options or prameters

click options are more difficult to provide with contextual defaults